app.rb

```diff
@@ -62,7 +75,7 @@ def auth_server_uri
     aal = params[:aal] || '2'
     skip_encryption = params[:skip_encryption]
     request_url = request.create(saml_settings(ial: ial, aal: aal))
-    request_url += "&skip_encryption=#{skip_encryption}" if skip_encryption
+    request_url += { skip_encryption: skip_encryption }.to_query if skip_encryption
```

Comment:
You lose the ampersand here :-(

Reply:
Yup! Got it in 4881b4d59224b2b7a8bee44bc9662c0cfb93b8ed